packages/tenant-process/src/services/tenantService.ts

```diff
@@ -87,8 +97,38 @@ export function tenantServiceBuilder(
         })
       );
     },
+
+    async updateTenantMails({
```

Comment:
the function is called `updateTenantMails` but
- is invoked in a generic `updateTenant` endpoint
- updates also the `kind` field

it probably needs a rename

Reply:
Renamed